### DISCUSSION TITLE
2011/07/03/%E6%BB%A1%E4%B8%83%E4%B8%AA%E6%9C%88%E7%9A%84%E8%B1%86%E8%B1%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#335] [Announcements]
# 2011/07/03/%E6%BB%A1%E4%B8%83%E4%B8%AA%E6%9C%88%E7%9A%84%E8%B1%86%E8%B1%86

再过几天豆豆就满七个月了，他的第四颗牙也终于长出来了。

https://blog.qizhen.xyz/2011/07/03/%E6%BB%A1%E4%B8%83%E4%B8%AA%E6%9C%88%E7%9A%84%E8%B1%86%E8%B1%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Ziwei
2011年07月8日 11:40 编辑
长得真帅气！ – Ziwei

回复
qiaoyiwei
2011年07月4日 15:27 编辑
兄弟，香蕉、芒果、西瓜等大寒水果盡量不給豆豆吃。
我家小寶就是在添加輔食的階段水果吃多了，傷了脾胃，影響了發育，幼兒園體檢身高和體重是班級倒數第二。
教訓啊！

回复
ruanqizhen
2011年07月5日 14:12 编辑
我们是蒸熟了吃的，应该还好。不过我也在考虑给他尝试点别的食品。

回复
